pillar/heroku/mitxonline.sls

```diff
@@ -84,6 +84,8 @@ heroku:
     MITX_ONLINE_SECURE_SSL_HOST: {{ env_data.MITXONLINE_SECURE_SSL_HOST }}
     MITX_ONLINE_USE_S3: True
     NODE_MODULES_CACHE: False
+    OPEN_EXCHANGE_RATES_APP_ID: {{ env_data.OPEN_EXCHANGE_RATES_APP_ID }}
```

Comment:
You'll need to also add the concrete value of the app ID in the dictionary at the top of the file.

Reply:
I don't have the APP ID